### ISSUE TITLE
[Bug]: a11y: Occupancy icons not visible or incorrectly displayed in High Contrast Mode

### ISSUE THREAD

**Issue by anastasiiapehl** [sbb-design-systems/lyne-components#4853]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Which version of Lyne Components are you using

4.12.0

### Operating system

Windows 11

### Input mode

None

### What happened?

When High Contrast Mode is enabled, the occupancy indicators are not displayed correctly.

<img width="945" height="374" alt="Image" src="https://github.com/user-attachments/assets/7df53adc-c84f-4b05-a676-e843029510db" />

**Expected Behavior:**
Occupancy indicators should be clearly visible in High Contrast Mode.

### Steps to reproduce the issue

High Contrast Mode in Windows 11: Settings > Accessibility > Contrast Themes.

### Relevant log output

```shell

```

**Comment by jeripeierSBB** [comment]
After research with @anastasiiapehl we isolated the problem to a corrupt user configuration in Chrome. After re-creating the configuration from scratch it started to work again. We decided to close this issue. If it ever happens again, feel free to re-open it.
